packages/api/src/controllers/access-control.ts

```diff
@@ -34,23 +39,38 @@ type GateConfig = {
 async function fireGateWebhook(
   webhook: DBWebhook,
   plabackPolicy: PlaybackPolicy,
-  accessKey: string
+  payload: AccessControlGatePayload,
+  isTrovoAuth: boolean
 ) {
   let timestamp = Date.now();
+  let jsonPayload = {
+    context: plabackPolicy.webhookContext,
+    accessKey: payload.accessKey,
+    timestamp: timestamp,
+  };
+
+  if (payload.webhookPayload) {
+    jsonPayload = { ...jsonPayload, ...payload.webhookPayload };
+  }
+
   let params = {
     method: "POST",
     headers: {
       "content-type": "application/json",
       "user-agent": "livepeer.studio",
     },
     timeout: WEBHOOK_TIMEOUT,
-    body: JSON.stringify({
-      context: plabackPolicy.webhookContext,
-      accessKey: accessKey,
-      timestamp: timestamp,
-    }),
+    body: JSON.stringify(jsonPayload),
   };
 
+  if (payload.webhookHeaders) {
+    params.headers = { ...params.headers, ...payload.webhookHeaders };
+  }
+
+  if (isTrovoAuth) {
+    params.headers["Trovo-Auth-Version"] = "1.1";
```

Comment:
This hurts 😆 
Let's make sure to migrate these to per-webhook headers configs later, which Trovo can play around however they want. We should avoid hardcoded logic for specific clients.